src/Http/Http.Extensions/gen/StaticRouteHandlerModel/Endpoint.cs

```diff
@@ -98,10 +102,13 @@ public Endpoint(IInvocationOperation operation, WellKnownTypes wellKnownTypes, S
     public List<Diagnostic> Diagnostics { get; } = new List<Diagnostic>();
 
     public (string File, int LineNumber, int CharacterNumber) Location { get; }
+#pragma warning disable RSEXPERIMENTAL002 // Experimental interceptable location API
+    public InterceptableLocation InterceptableLocation { get; }
+#pragma warning restore RSEXPERIMENTAL002
     public IInvocationOperation Operation { get; }
 
     public override bool Equals(object o) =>
-        o is Endpoint other && Location == other.Location && SignatureEquals(this, other);
+        o is Endpoint other && InterceptableLocation == other.InterceptableLocation && SignatureEquals(this, other);
```

Comment:
I don't think we implemented `operator ==` on these. That might be a bug on our side. I'll have to follow up.

Reply:
An explicit `==` implementation would be nice. I assume this does implement equality checks correctly though given that it does the correct things in our cache invalidation tests for incremental generators.